src/main/java/org/jenkinsci/plugins/mesos/api/InMemoryRepository.java

```diff
@@ -0,0 +1,28 @@
+package org.jenkinsci.plugins.mesos.api;
+
+import akka.Done;
+import com.mesosphere.usi.repository.PodRecordRepository;
+import scala.collection.Map$;
+import scala.collection.immutable.Map;
+import scala.concurrent.Future;
+import scala.concurrent.Future$;
+
+public class InMemoryRepository implements PodRecordRepository {
```

Comment:
added a change in #40 that works the same, except using java futures and converting to scala, that may be more readable

Reply:
#40 is much cleaner. Let's get that one out of the door first.